rust-ceramic/src/docker.md

```diff
@@ -0,0 +1,8 @@
+# Docker
+
+Docker images for rust-ceramic are available from our AWS container registry.
+
+Currently, the latest release is `0.12.0`.
```

Comment:
I'd prefer to to have to update this every time we cut a release. Can we either automate updating or word it so that we can just point at `latest` tag instead of the name of the release? 

Note the release script does have the ability to update this automatically we would just need to tell it about this file.

Reply:
Now requires https://github.com/3box/rust-builder/pull/20

So that we can update the version number during a PR.